ee/src/main/java/org/jboss/as/ee/subsystem/ManagedScheduledExecutorServiceResourceDefinition.java

```diff
@@ -46,6 +48,12 @@
  */
 public class ManagedScheduledExecutorServiceResourceDefinition extends SimpleResourceDefinition {
 
+    /**
+     * the resource definition's dynamic runtime capability
+     */
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.ee.concurrent.scheduledexecutor", true, ManagedScheduledExecutorServiceAdapter.class)
```

Comment:
Can we not use javax.enterprise.concurrent.ManagedScheduledExecutorService for the service type? Generally, we should avoid exposing specific implementation classes via the capability contract.